### PR TITLE
Fixing Arduino and raspberry pi agent downloading issue

### PR DESCRIPTION
**Pull request by lasanthaDLPDS**
… user

### CODE REVIEW

File: components/device-types/arduino-plugin/org.wso2.carbon.device.mgt.iot.arduino.api/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/service/impl/ArduinoServiceImpl.java [hunk 1]
```diff
@@ -221,13 +221,21 @@ private ZipArchive createDownloadFile(String owner, String deviceName)
         String applicationUsername = PrivilegedCarbonContext.getThreadLocalCarbonContext().getUserRealm()
                 .getRealmConfiguration().getAdminUserName() + "@" + PrivilegedCarbonContext
                 .getThreadLocalCarbonContext().getTenantDomain();
-        ;
         if (apiApplicationKey == null) {
+            String adminUsername = PrivilegedCarbonContext.getThreadLocalCarbonContext().getUserRealm().getRealmConfiguration().getAdminUserName();
```
Comment (madhawap) [inline components/device-types/arduino-plugin/org.wso2.carbon.device.mgt.iot.arduino.api/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/service/impl/ArduinoServiceImpl.java]: This line seems exceeding the right margin column which should be 110. Please reformat it.

File: components/device-types/raspberrypi-plugin/org.wso2.carbon.device.mgt.iot.raspberrypi.api/src/main/java/org/wso2/carbon/device/mgt/iot/raspberrypi/service/impl/RaspberryPiServiceImpl.java [hunk 2]
```diff
@@ -203,14 +203,23 @@ private ZipArchive createDownloadFile(String owner, String deviceName, String sk
             throw new DeviceManagementException(msg);
         }
         if (apiApplicationKey == null) {
+            String adminUsername = PrivilegedCarbonContext.getThreadLocalCarbonContext().getUserRealm().getRealmConfiguration().getAdminUserName();
```
Comment (madhawap) [inline components/device-types/raspberrypi-plugin/org.wso2.carbon.device.mgt.iot.raspberrypi.api/src/main/java/org/wso2/carbon/device/mgt/iot/raspberrypi/service/impl/RaspberryPiServiceImpl.java]: This line also seems exceeding the right margin column which should be 110. Please check the other lines of code and reformat them.

File: components/device-types/arduino-plugin/org.wso2.carbon.device.mgt.iot.arduino.api/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/service/impl/ArduinoServiceImpl.java (hunk 1 above)
Comment (lasanthaDLPDS) [reply components/device-types/arduino-plugin/org.wso2.carbon.device.mgt.iot.arduino.api/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/service/impl/ArduinoServiceImpl.java]: This is fixed b2c7033608104e6a154a2cfd8b6fa1df741e1676

File: components/device-types/raspberrypi-plugin/org.wso2.carbon.device.mgt.iot.raspberrypi.api/src/main/java/org/wso2/carbon/device/mgt/iot/raspberrypi/service/impl/RaspberryPiServiceImpl.java (hunk 2 above)
Comment (lasanthaDLPDS) [reply components/device-types/raspberrypi-plugin/org.wso2.carbon.device.mgt.iot.raspberrypi.api/src/main/java/org/wso2/carbon/device/mgt/iot/raspberrypi/service/impl/RaspberryPiServiceImpl.java]: This is fixed b2c7033608104e6a154a2cfd8b6fa1df741e1676

**Pull request merged**